eng/tools/mcp/azure-sdk-java-mcp/src/index.ts

```diff
@@ -30,12 +30,12 @@ server.registerTool(
     "clean_java_source",
     {
         description:
-            "Remove all generated Java source files and directories for a given module. This tool is typically used to clean up the output of previous SDK generations before a new build.",
+            "Remove all generated Java source files and directories for a given module. This tool is typically used to clean up the output of previous SDK generations before a new build. It should only be applied to packages with the prefix `azure-resourcemanager-*`.",
```

Comment:
shoud we add the check inside the tool？ https://github.com/Azure/azure-sdk-for-java/blob/main/eng/tools/mcp/azure-sdk-java-mcp/src/clean-java-source.ts